substratevm/src/com.oracle.svm.jvmtiagentbase/src/com/oracle/svm/jvmtiagentbase/Support.java

```diff
@@ -182,6 +182,15 @@ public static JNIObjectHandle getObjectArgument(int slot) {
         return handlePtr.read();
     }
 
+    public static int getIntArgument(int slot) {
+        CIntPointer handlePtr = StackValue.get(CIntPointer.class);
+        JvmtiError error = jvmtiFunctions().GetLocalInt().invoke(jvmtiEnv(), nullHandle(), 0, slot, handlePtr);
+        if (error != JvmtiError.JVMTI_ERROR_NONE) {
+            throw new RuntimeException(error.toString());
```

Comment:
This should be consistent with the other methods and not throw an exception. It could be solved by passing in a `CIntPointer` from the caller and returning a `boolean` whether it was successful. Also, the variable should not be called `handlePtr` since it does not contain a handle.

Reply:
Whole method is removed